ohsome/response.py

```diff
@@ -78,7 +82,25 @@ def _as_geodataframe(self, multi_index=True):
             return gpd.GeoDataFrame(crs="epsg:4326", columns=["@osmId", "geometry"])
 
         try:
+            if explode_tags is not None:
```

Comment:
I think you do not necessarily need the `is not None`

Reply:
I do because the default is an empty `tuple` and that evaluates to `False`

Reply:
ah ok